modules/navier_stokes/doc/content/source/linearfvkernels/LinearFVScalarAdvection.md

```diff
@@ -0,0 +1,23 @@
+# LinearFVScalarAdvection
```

Comment:
The scalar advection schemes don't use the density. We will need some modifications in the RC object to be able to obtain the face velocities (volumetric flow rates).

Reply:
very true. this should be fixed now. That was a big oversight on my part, what was I thinking